Assets/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerController : MonoBehaviour
+{
+    public static PlayerController instance;
+
+    public float moveSpeed;
```

Comment:
nit: can you group and comment the variables by category. ex:
// Movement Variables
moveSpeed;
moveVelocity;

Reply:
kk sounds good